mm2src/coins/rpc_command/tendermint/staking.rs

```diff
@@ -148,3 +149,16 @@ pub async fn validators_rpc(
         validators: validators.into_iter().map(jsonize_validator).collect(),
     })
 }
+
+#[derive(Clone, Debug, Deserialize)]
+pub struct DelegatePayload {
+    pub validator_address: String,
+    pub fee: Option<WithdrawFee>,
```

Comment:
Should we use `WithdrawFee` here if this is used for cosmos only? I understand that it's to use this function https://github.com/KomodoPlatform/komodo-defi-framework/blob/0f08e053515ce3360db5a7199b590ebf21fc6af6/mm2src/coins/tendermint/tendermint_coin.rs#L2226 plus the `WithdrawFee` type is already used from GUI so my opinion is to leave it. But this is another addition that will add to the list of things to refactor to separate different coins code from each other. Just opening this for discussion if you have an easy way to solve it, otherwise leave it as is.

Reply:
> already used from GUI so my opinion is to leave it.

That's the only reason I used it that way.

> But this is another addition that will add to the list of things to refactor to separate different coins code from each other. Just opening this for discussion if you have an easy way to solve it, otherwise leave it as is.

As it's already used for `withdraw` I am not sure if it adds an additional cost to existing problem.